pkg/kubectl/resource/builder.go

```diff
@@ -325,20 +329,6 @@ func hasCombinedTypeArgs(args []string) (bool, error) {
 	}
 }
 
-// ResourceTypeAndNameArgs expects two arguments, a resource type, and a resource name. The resource
-// matching that type and and name will be retrieved from the server.
-func (b *Builder) ResourceTypeAndNameArgs(args ...string) *Builder {
```

Comment:
`ResourceTypeOrNameArgs` is literately a superset of this. Which is probably why this is not used anywhere now.